packages/applications/notifications/src/subscribers/lauréat/représentantLégal.notification.ts

```diff
@@ -0,0 +1,65 @@
+import { Message, MessageHandler, mediator } from 'mediateur';
+
+import { IdentifiantProjet } from '@potentiel-domain/common';
+import { Option } from '@potentiel-libraries/monads';
+import { Event } from '@potentiel-infrastructure/pg-event-sourcing';
+import {
+  CandidatureAdapter,
+  récupérerPorteursParIdentifiantProjetAdapter,
+} from '@potentiel-infrastructure/domain-adapters';
+import { ReprésentantLégal } from '@potentiel-domain/laureat';
+import { Routes } from '@potentiel-applications/routes';
+
+import { EmailPayload, SendEmail } from '../../sendEmail';
+
+export type SubscriptionEvent = ReprésentantLégal.ReprésentantLégalEvent & Event;
+
+export type Execute = Message<'System.Notification.Lauréat.ReprésentantLégal', SubscriptionEvent>;
+
+const templateId = {
+  modifier: 6502927,
```

Comment:
```suggestion
  modifierReprésentantLégal: 6502927,
```

(j'ai regardé sur d'autres domaines, on spécifie un peu plus)